platform/00-kubernetes-minikube-setup.sh

```diff
@@ -113,7 +113,7 @@ case "${PLATFORM}" in
         echo "Actual SHA256 for $TKN_FILE_NAME: $ACTUAL_SHA256"
         exit 1
       )
-      sudo tar xvzf $TKN_FILE_NAME -C /usr/local/bin tkn
+      sudo tar xvzf $TKN_FILE_NAME -C $INSTALL_DIR tkn
```

Comment:
In the future, I'd like to make `$INSTALL_DIR` user configurable, so I recommend quoting it.

```suggestion
      sudo tar xvzf $TKN_FILE_NAME -C "$INSTALL_DIR" tkn
```